src/services/documentService.ts

```diff
@@ -37,6 +40,36 @@ export default class DocumentService {
     });
   }
 
+  public async signDocument(
+    unsignedContentBase64: string
+  ): Promise<Either<Error, string>> {
+    try {
+      const soapClient: soap.Client = await soap.createClientAsync(
```

Comment:
may we create the soap client a a singleton and pass it to the documentService as param ?